.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature
```

Comment:
and here "feature request" or such?

Reply:
Maybe we need the one label that is "incoming", because accurate classification (bug, feature, enhancement, ...) can only happen by us.

Reply:
I created `Needs Triage`